config/dev.cfg

```diff
@@ -16,7 +16,8 @@ credential.password=testPwd
 
 [path]
 path.ebooks=ebooks
-path.extras=ebooks/extras
+path.extras=extras
+path.separate=true
```

Comment:
The first package of improvements: 1c557be :smiley: 
I do not promise when I do the rest. I will try to do it in my free time.